content/en/docs/performance/resource_consumption.md

```diff
@@ -0,0 +1,151 @@
+---
+draft: false
+linktitle: Kmesh resource consumption
+menu:
+  docs:
+    parent: performance
+    weight: 2
+title: Kmesh resource consumption
+toc: true
+type: docs
+---
+# 1. Background Information
+eBPF programs consume a certain amount of CPU and memory resources during their execution, which may affect the performance of other applications. To gain a deeper understanding of Kmesh’s resource consumption under different loads, we conducted several CPU and memory stress tests. These tests aim to evaluate the limits of CPU and memory consumption of Kmesh in real-world usage scenarios.
+
+**This documentation is based on Kmesh 0.4 ads mode**
+# 2. Environment setup
+![resource_env](images/resource_test_env.png)
```

Comment:
Still not able to show https://deploy-preview-67--kmesh-net.netlify.app/en/docs/performance/resource_consumption/#31-deploy-1-app-a-and-set-app-a-pods-cpu-limitthen-generage-load-and-collect-kmesh-ebpf-cpu-usage

Please refer to other docs

Reply:
I changed the image path, but the image still doesn't show up